src/easylink/implementation_metadata.yaml

```diff
@@ -1,194 +1,193 @@
-# development dummies
 step_1_python_pandas:
-  steps: 
-    - step_1
+  steps:
```

Comment:
Nothing new in here, it's just that when registering new containers it changes the indenting and I didn't want that to surprise you the first time you use the tool